README.md

```diff
@@ -1 +1,113 @@
-# terraform-cloud-connect
\ No newline at end of file
+# terraform-cloud-connect
+
+## Description
+
+The main purpose of this script is to create an IAM User with enough permission and a Security Group with correct Inbound and Outbound rules so that Bitmovin's Cloud Connect product will be able to run Video Encodings in the configured infrastructure.
+
+The Terraform Cloud Connect module does the following:
+
+- Creates new IAM user
+- Assigns required permission required for Cloud-Connect: ~ AmazonEC2FullAccess
+- Creates a Security Group
+- Adds required Inbound and Outbound rules
+- Outputs:
+    - Account Id
+    - Access Key
+    - Secret Access Key
+    - Security Group Id
+
+The outputs that are produced by the script can be used in a next step to configure the Bitmovin Encoder for Cloud Connect.
+
```

Comment:
i would also add a section about the provider set-up and mention the regional configuration

Reply:
Done

Reply:
Done